### PR TITLE
Openvpn

### PR DESCRIPTION
**Pull request by shapi78**
Added bastion host support, using firewall module and openvpn module

### CODE REVIEW

**Review by alvagante — COMMENTED** [review]
As already discussed in call, this looks fine to me.

File: hieradata/role/openvpn.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  net.ipv4.ip_forward: 1
+  net.ipv4.conf.default.rp_filter: 0
+  net.ipv4.conf.all.rp_filter: 0
+  net.ipv4.conf.all.send_redirects: 0
+  net.ipv4.conf.default.send_redirects: 0
+  net.ipv4.icmp_ignore_bogus_error_responses: 1
+  net.ipv4.conf.default.log_martians: 0
+
+openvpn::servers:
+  'winterthur':
```
Comment (alvagante) [inline hieradata/role/openvpn.yaml]: winthertur here is just a name, can be anything (for example graduway), just keep the same nem here and in line 32

File: hieradata/role/openvpn.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+psick::profiles::linux_classes:
+  openvpn: openvpn
+  fw: psick::firewall::iptables
+
+
+psick::firewall::iptables::extra_rules:
```
Comment (alvagante) [inline hieradata/role/openvpn.yaml:7]: If you don't want to manage ipv6 add somewhere here:
psick::firewall::iptables::manage_ipv6: false

Remember to update the psick module to latest version.

Comment (shapi78) [reply hieradata/role/openvpn.yaml:7]: done with r10k command on the puppet server

**Pull request merged**